src/templates/best-cards/index.html

```diff
@@ -18,27 +18,49 @@ <h1>{{ _('Best Cards') }}</h1>
     </p>
     {% endtrans %}
     <h2>{{ _('Generic') }}</h2>
+	{% trans trimmed
+		master=link("/best-cards/generic/master-cards"),
+		political=link("/best-cards/generic/political-actions"),
+		no_discipline=link("/best-cards/generic/no-discipline"),
+		animalism=link("/best-cards/generic/animalism"),
+		auspex=link("/best-cards/generic/auspex"),
+		celerity=link("/best-cards/generic/celerity"),
+		dominate=link("/best-cards/generic/dominate"),
+		fortitude=link("/best-cards/generic/fortitude"),
+		necromancy=link("/best-cards/generic/necromancy"),
+		obfuscate=link("/best-cards/generic/obfuscate"),
+		potence=link("/best-cards/generic/potence"),
+		presence=link("/best-cards/generic/presence")
+	%}
```

Comment:
Well, sure, if you think this is more consistent.

However, there is nothing to translate here, so I fear we'll see useless HTML tags in the translations file.
Also, I added the disciplines in the Jinja context to be able to use `{{ aus }}` instead of `<i>a</i>`, in order to avoid too many tags in the translated sections.